network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -438,6 +439,58 @@ class ParticipateHighlights2(ParticipateHighlightsBase):
     )
 
 
+@register_snippet
+class FocusArea(models.Model):
+    interest_icon = models.ForeignKey(
+        'wagtailimages.Image',
```

Comment:
Will this always be an image, never an svg?

Reply:
yeah, given the trouble with SVG, we only use static asset svg icons, but jpg/png works fine here, too, and is easier to update for staff.